README.md

```diff
@@ -1,102 +1,139 @@
 
-Ansible role for managing Advanced Intrusion Detection Environment (AIDE).
+This is an ansible role that installs and configures the [Advanced Intrusion Detection Environment (AIDE)](https://aide.github.io). For Day 2 tasks it can run integrity checks and update the AIDE database.
+
+_Notice:_ This is a very early stage of a work in progress. Please use with
+extreme caution as it might break your system.
+
+## What does this role do for you?
+
+* It ensures that the `aide` package is installed on the remote nodes
+* As an optional task it can generate the `/etc/aide.conf` file and template it out to the remote nodes
+* It initializes the AIDE database
+* The AIDE databases from the remote nodes are stored in a central directory on the controller node
+* It runs AIDE integrity checks on the remote nodes
+* It updates the AIDE databases and stores them on the controller node
+
+## How does the role do that?
+
+* The role is controlled by using role variables
+* If you run the playbook without specifying any role variable the role will change nothing on your remote nodes
+* To execute some supported use cases you need to explicitly specify one or more of the following variables
+
+### Available tags to control and use the role
+
+## What does this role not do for you?
+
+* It does not explain how to create a good AIDE configuration that suits your requirements; that task remains for you to accomplish
 
 ## Requirements
 
-Any prerequisites that may not be covered by Ansible itself or the role should
-be mentioned here.  This includes platform dependencies not managed by the
-role, hardware requirements, external collections, etc.  There should be a
-distinction between *control node* requirements (like collections) and
-*managed node* requirements (like special hardware, platform provisioning).
+This role has no special requirements as it uses `ansible.builtin` modules
+only.
+
+## Role Variables
+
+### aide_custom_template
```

Comment:
I am thinking of renaming this variable, because custom is self-explain in this context. How about `aide_db_template`?

Reply:
renamed